app/components/ui/HamburgerMenu.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import { push as Menu } from 'react-burger-menu';
+
+import styles from './HamburgerMenu.scss';
+
+
+const burgerStyles = {
+  bmBurgerButton: {
+    display: 'none',
+  },
+  bmCrossButton: {
+    display: 'none',
+  },
+  bmMenu: {
+    padding: '0',
+    borderLeft: '1px solid #f4f4f4',
+  },
+  bmOverlay: {
+    display: 'none',
+  },
+};
+
+const links = [
+  { to: '/', text: 'Home' },
+  { to: 'https://sheltertech.org/volunteer', text: 'Volunteer' },
+  { to: 'mailto:info@sheltertech.org', text: 'Email Us' },
+  { to: 'https://www.facebook.com/ShelterTechOrg/', text: 'Facebook' },
+  { to: 'https://twitter.com/sheltertechorg', text: 'Twitter' },
+  { to: '/terms-of-service', text: 'Terms of Service' },
+  { to: '/privacy-policy', text: 'Privacy Policy' },
+];
+
+const HamburgerMenu = ({
+  isOpen,
+  location,
+  onStateChange,
+  outerContainerId,
+  pageWrapId,
+  toggleHamburgerMenu,
+}) => (
+  <Menu
+    isOpen={isOpen}
+    onStateChange={onStateChange}
+    outerContainerId={outerContainerId}
+    pageWrapId={pageWrapId}
+    right
+    styles={burgerStyles}
+    width="275px"
```

Comment:
Why is this random style here and not in the burgerStyles above?

Reply:
Once again, it's part of the react-burger-menu API, and I think they did it this way because this one width affects multiple of the subcomponents, since you want the width of the side menu, the distance to translate the side menu when opening it, and the distance to translate the rest of the page to all be the same.

Reply:
Ah. I see. Makes sense.